tmk_core/common/eeconfig.h

```diff
@@ -57,6 +58,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // EEHANDS for two handed boards
 #define EECONFIG_HANDEDNESS                     		(uint8_t *)11
 #define EECONFIG_KEYBOARD                          (uint32_t *)12
+#define EECONFIG_VELOCIKEY                          (uint8_t *)13
```

Comment:
This cannot go here. It will overlap.

This NEEDS to be:
```suggestion
#define EECONFIG_VELOCIKEY                          (uint8_t *)16
```